microshift_updating/microshift-version-updating.adoc

```diff
@@ -0,0 +1,19 @@
+:_content-type: ASSEMBLY
+[id="microshift-version"]
+= Checking which version you have installed
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-version-update
+
+toc::[]
+
+Before attempting and update, determine which version of {product-title} you have installed.
+
+Updates are only available from:
```

Comment:
```suggestion
The following upgrade paths are supported:
```

Or something else. If you opt to keep something similar to what you have, I think it'll need slightly revised to adhere to IBMSG's guidance on colons and complete sentences: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-colons#introductory-text